website/src/content/docs/docs/standard-library/pagination.md

```diff
@@ -76,6 +76,22 @@ It is possible to use server driven pagination on top of client driven paginatio
 };
 ```
 
+In this case, the body of the server response for a page has a property marked as continuation token, which can be used to fetch the next page. When fetching the next page, request will include the continuation token as a query parameter.
+
+The continuation token can be in other locations as well, for instance, the below spec indicates that the server response for a page can have a header containing the continuation token. When fetching the next page, request will include the continuation token as a header parameter.
```

Comment:
It may be confusing that you use both a header from response and as a request parameter. We don't want people to get confused, that there is a correlation: there isn't. You can get the continuation token from a header on the response side, and inject it back as a query.

Reply:
Good point, make sense, let me update.